Dockerfile

```diff
@@ -1,31 +1,18 @@
 # This file uses a multi-stage build strategy. The build stage sets up the nvm environment and builds configurator, while the second stage copies this into a clean container without any build tools.
 
 ## First Stage- Build
-FROM nginx as build
+FROM node:14 as build
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-# Pre-reqs
-COPY conf/yarn.list /tmp
-RUN apt update && \
-    apt install --no-install-recommends -y \
-        ca-certificates \
-        curl \
-        gnupg \
-        nodejs && \
-    mv /tmp/yarn.list /etc/apt/sources.list.d/yarn.list && \
-    curl https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
-    apt update -o Dir::Etc::sourcelist=/etc/apt/sources.list.d/yarn.list && \
-    apt install --no-install-recommends -y \
-        yarn && \
-    rm -rf /var/lib/apt/lists/*
-RUN curl https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash
-
 # Copy files into place
 COPY . /qmk_configurator/
+WORKDIR /qmk_configurator/
 
 # Build configurator
-RUN /bin/bash -i /qmk_configurator/bin/docker_build.sh
+RUN yarn install
+ENV VUE_APP_API_URL=/api
+RUN yarn run build
```

Comment:
There is still a change in behavior here. What you've done is set the environment variable `VUE_APP_API_URL` for all subsequent commands, including the app itself, not just the build. This may or may not be ok, @yanfali would need to weigh in on that.

Reply:
This line is the last command for this container build. The following line starts a new container build (this is a multistage build), so this `ENV` does not apply to anything else.